agent/tests/test_memcached.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "tlib_php.h"
+#include "tlib_datastore.h"
+
+#include "php_agent.h"
+#include "php_memcached.h"
+#include "util_system.h"
+
+tlib_parallel_info_t parallel_info
+    = {.suggested_nthreads = -1, .state_size = 0};
+
+static char* system_host_name;
+
+static void test_create_datastore_instance(void) {
+  /*
+   * Test : Normal operation.
+   */
```

Comment:
I'm not sure which tests this comment applies to first one or all? If only the first one, then each subsequent should have a similar comment, which IMHO is redundant because the first argument passed to `assert_datastore_instance_equals_destroy` serves the same purpose. If all tests, then it is not accurate, because the last test that passes `NULL` as hostname does not seem Normal operation.
```suggestion

```

Reply:
The comment would be useful to group tests by the value of `port` passed to `nr_php_memcached_create_datastore_instance`. I.e. if `port == 0`, then first argument of `nr_php_memcached_create_datastore_instance` is used for `port_path_or_id` in the datastore instance created. If `port != 0` then first argument is used for `host` and second argument is used as `port_path_or_id`.

Reply:
organized and consolidated unit tests in [5597d7f](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/5597d7fccf8657254c4c9d486612ef821b2754be)

Reply:
removed comment in [62b8889](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/62b888920b6a143f6c3d6dad923ad1fa61951af8)